style.css

```diff
@@ -0,0 +1,121 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+.container {
+  width: 100%;
+  height: 100vh;
+  position: relative;
+  overflow: hidden;
+}
+.container .slider {
+  width: 500%;
+  height: 100%;
+  position: absolute;
+  inset: 0 0;
+  float: left;
+  animation: slide 15s linear infinite;
+}
+.container .slider .slide {
+  position: relative;
+  width: 20%;
+  height: 100%;
+  float: left;
+}
+.container .slider .slide .caption {

```

Comment:
create space between group similar property

Reply:
Would you help me with a "suggestion" how you would have grouped? It will be helpful for me.

Reply:
` position: absolute;
  bottom: 60px;
  left: 60px;
  right: 60px;

  padding: 30px;

  background: rgba(0, 0, 0, 0.5);
  
  box-sizing: border-box;

  transition: 5s;`

Reply:
` position: absolute;
  bottom: 60px;
  left: 60px;
  right: 60px;

  padding: 30px;

  background: rgba(0, 0, 0, 0.5);
  
  box-sizing: border-box;

  transition: 5s;`

Reply:
`` position: absolute;
  bottom: 60px;
  left: 60px;
  right: 60px;

  padding: 30px;

  background: rgba(0, 0, 0, 0.5);
  
  box-sizing: border-box;

  transition: 5s;

```

Reply:
Thanks brother wil do change it accordingly